backend/base.py

```diff
@@ -530,6 +530,11 @@ def goalsUpdate(): # pragma: no cover
         "email": current_user,
     }
     try:
+        current_user = get_jwt_identity()
+        profile = mongo.user.find_one({"email": current_user})
```

Comment:
Is it possible to use the "query" variable which is set to the same as what is being queried?

Reply:
smart